Userland/Libraries/LibWeb/HTML/StructuredSerialize.cpp

```diff
@@ -397,7 +400,11 @@ class Serializer {
                 }
             }
 
-            // FIXME: 3. Otherwise, if value is a platform object that is a serializable object, then perform the serialization steps for value's primary interface, given value, serialized, and forStorage.
+            // 3. Otherwise, if value is a platform object that is a serializable object, then perform the serialization steps for value's primary interface, given value, serialized, and forStorage.
+            else if (value.is_object() && is<Bindings::Serializable>(value.as_object())) {
+                auto& serializable = dynamic_cast<Bindings::Serializable&>(value.as_object());
```

Comment:
Since you've already verified the type with `is<Bindings::Serializable>()`, we can use `static_cast` instead of `dynamic_cast`.

Reply:
`Bindings::Serializable` is a polymorphic class (with virtual functions) so I am forced to use `dynamic_cast` here.

Reply:
!!